### ISSUE TITLE
The component is failing to load in IE9 browser

### ISSUE THREAD

**Issue by dPramod** [raythree/react-bootstrap-table#3]
This is a wonderful component but not able to work with IE9..

**Comment by raythree** [comment]
Hi, yes, sorry about that, but it is using all the latest tools and I made the decision not to support IE earlier than 11. I have a Windows VM that has IE 11 for testing and it works in that. When I set compatibility mode to IE9, I see where an exception is being generated. Unfortunately it is inside code that was transpiled from es6 using babel, so I don't think there is an easy fix.

Regards,
Bill

**Comment by dPramod** [comment]
Thank  you Bill for looking into this !! Is there any other way we can have a patch to fix the issue on IE9/10. We need to support IE browsers and this being caught at the last min, changing the complete library can be quite tough. Please suggest if there are any alternate approaches.

**Comment by raythree** [comment]
Yes. If you clone the repo and execute "npm run build" this runs the src through babel and produces the distribution in the lib directory. If you look at the generated code, you will see this in BootstrapTable constructor:
```
var BootstrapTable = function (_Component) {
  _inherits(BootstrapTable, _Component);

  function BootstrapTable(props, context) {
    _classCallCheck(this, BootstrapTable);

    var _this = _possibleConstructorReturn(this, Object.getPrototypeOf(BootstrapTable).call(this, props, context));
```
The ```_this``` object returned seems to be null under IE9. You might be able to try different babel presets (see the .babelrc file) to get it to work.

**Comment by big5jeff** [comment]
I forked the repo and added an ie polyfill dependency for loose settings on es2015 transpiling. See https://github.com/Big5IO/reactjs-bootstrap-table-ie9
